src/test/scala/com/gu/hmac/HMACHeadersTest.scala

```diff
@@ -1,34 +1,47 @@
 package com.gu.hmac
 
 import java.net.URI
-
 import org.joda.time.DateTime
-import org.scalatest.{FlatSpec, Matchers}
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.flatspec.AnyFlatSpec
+
+import java.time.{Clock, Instant, ZoneId}
 
-class HMACHeadersTest extends FlatSpec with Matchers {
+class HMACHeadersTest extends AnyFlatSpec with Matchers {
   import HMACDate.DateTimeOps
+  val validSecret = "secret"
+  val invalidSecret = "wrong"
+
+  val uri = new URI("http:///www.theguardian.com/signin?query=someData")
+  val date = new DateTime(1994, 11, 15, 8, 12)
+
+  val expectedHMAC = "3AQ08uT4ToOISOXWMr68UvzrgrqIx3KK/pKEenwVES8="
```

Comment:
This value should be hard-coded and not generated as part of the test. In this way we can ensure that changes in code here do not result in different tokens being generated (this token was generated before the clock updates in `CreateHMACHeader` and used to verify that change did not cause different token value generation).